source/user_guide/advanced_capabilities/feature_importance.rst

```diff
@@ -51,53 +51,51 @@ The created :class:`~Trainee` will be referenced as ``trainee`` in the sections
-and setting them to ``True`` will cache the desired metrics. For example, ``mda_robust`` and ``contributions_robust`` will calculate the robust versions of MDA and Feature Contributions, while ``mda`` and ``contributions`` will calculate the non-robust versions.
+To get global feature importance metrics, :py:meth:`Trainee.react_aggregate`, is called on a trained and analyzed Trainee. :py:meth:`Trainee.react_aggregate` calls react internally on the cases already trained into the Trainee and calculates the metrics. In this method, the desired metrics can be selected as parameters. These parameters are named individually
+in the ``details`` parameter and setting them to ``True`` will calculate and return the desired metrics. For example, ``feature_mda_robust`` and ``feature_contributions_robust`` will calculate the robust versions of MDA and Feature Contributions, while ``feature_mda_full`` and ``feature_contributions_full`` will calculate the non-robust (full) versions.
+An action feature must be specified. ``feature_influences_action_feature`` is recommended for feature influence metrics such as feature contributions and mda, especially when used in conjuction with retrieving prediction stats, however, ``action_feature`` can be also used as well. ``action_feature`` sets the action feature for both influence metrics and prediction stats. Since often
+only the influence metrics's action feature is intended to be set, ``feature_influences_action_feature`` provides a more precise parameter.
 
 .. code-block:: python
 
-    t.react_into_trainee(
+    feature_contributions_robust = trainee.react_aggregate(
         context_features=context_features,
-        action_feature=action_features[0],
-        contributions_robust=True
+        feature_influences_action_feature=action_features[0],
+        details={"feature_contributions_robust" : True}
     )
 
-
-In order to extract the metrics, :py:meth:`Trainee.get_prediction_stats` is called. An action feature must be specified, and the ``stats`` parameter is used determine which metrics to return. The ``stats`` parameter takes a list, so multiple
-metrics may be specified together, but for this example they are separated. If robust metrics are calculated, then the ``robust`` parameter must be set to ``True`` to retrieve these metrics. If non-robust metrics are calculated, then the ``robust`` parameter can be set to the default value.
-
-.. code-block:: python
-
-    robust_feature_contributions = t.get_prediction_stats(action_feature=action_features[0], robust=True, stats=['contribution'])
-    robust_feature_mda = t.get_prediction_stats(action_feature=action_features[0], robust=True, stats=['mda'])
+    feature_mda_robust = trainee.react_aggregate(
+        context_features=context_features,
+        feature_influences_action_feature=action_features[0],
+        details={"feature_mda_robust" : True}
```

Comment:
The other edits in this PR use single quotes, I'm more partial to double quotes personally but we should at least keep it consistent.

```suggestion
        details={'feature_mda_robust' : True}
```